src/Illuminate/View/Component.php

```diff
@@ -56,6 +55,7 @@ public function resolveView()
     /**
      * Create a Blade view with the raw component string content.
      *
+     * @param  \Illuminate\View\Factory  $factory
```

Comment:
This should probably be the contract, not the implementation.

Reply:
Right thanks.